WorkloadManagementSystem/Agent/SiteDirector.py

```diff
@@ -809,6 +809,12 @@ def _getPilotOptions( self, queue, pilotsToSubmit ):
     # diracVersion is a list of accepted releases
     pilotOptions.append( '-r %s' % ','.join( str( it ) for it in diracVersion ) )
 
+    #lcgBundle defined?
+    lcgBundleVersion = opsHelper.getValue( "Pilot/lcgBundleVersion", "" )
```

Comment:
I think that the convention is to start option names with a capital letter. Am I a hated purist ?